NeuroID/NeuroIDClass/Extensions/NIDAdvancedDevice.swift

```diff
@@ -90,17 +90,28 @@ public extension NeuroID {
         NeuroID.saveEventToLocalDataStore(nidEvent)
     }
     
+    internal static func captureLOGEvent(_ message: String, level: String) {
+        let logEvent = NIDEvent(type: .log)
+        logEvent.m = message
+        logEvent.level = level
+        
+        NeuroID.saveEventToLocalDataStore(logEvent)
+    }
+    
```

Comment:
probably un-needed at this point since its only used once and we can make the event in-line in one spot?